cvrf2csaf/cvrf2csaf.py

```diff
@@ -65,7 +65,7 @@ def _validate_and_open_file(cls, file_path):
         try:
             xml_objectified = objectify.parse(file_path, parser).getroot()
         except etree.ParseError as e:
-            logging.critical(f"Document not valid: {e}.")
+            logging.critical(f'Document not valid: {e}.')
```

Comment:
This is related to the discussion in #24. I'm happy to get that merged as it is - just flagging that we might need to revisit it later.